telescope/external.py

```diff
@@ -182,32 +188,35 @@ def _wait_for_results_chunk(self, job_id, page_token):
     Returns:
       (list) A list of result rows in the current chunk of results.
     """
-    MAX_RESULTS_PER_GET = 100000
-    query_request = {'projectId': self._project_id,
-                     'jobId': job_id,
-                     'maxResults': MAX_RESULTS_PER_GET,
-                     'timeoutMs': 0}
-    if page_token is not None:
-      query_request['pageToken'] = page_token
-
-    retries_remaining = 4
-    while True:
-      try:
-        return self._execute_job_query(query_request)
-      except BigQueryCommunicationError as e:
-        if retries_remaining > 0:
-          sleep_seconds = 10
-          logging.warning(('Failed to communicate with BigQuery to retrieve '
-                           'job %s. Retrying in %d seconds... (%d attempts '
-                           'remaining)'),
-                          job_id, sleep_seconds, retries_remaining)
-          time.sleep(sleep_seconds)
-          retries_remaining -= 1
-        else:
-          raise e
-
-  def _execute_job_query(self, bq_query):
-    """Executes a query to retrieve BigQuery query results.
+        MAX_RESULTS_PER_GET = 100000
+        query_request = {
+            'projectId': self._project_id,
+            'jobId': job_id,
+            'maxResults': MAX_RESULTS_PER_GET,
+            'timeoutMs': 0
+        }
+        if page_token is not None:
+            query_request['pageToken'] = page_token
+
+        retries_remaining = 4
+        while True:
+            try:
+                return self._execute_job_query(query_request)
+            except BigQueryCommunicationError as e:
+                if retries_remaining > 0:
+                    sleep_seconds = 10
+                    logging.warning(
+                        ('Failed to communicate with BigQuery to retrieve '
```

Comment:
Same as the case above. We can probably do line breaks better with the changed indent.